spinta/utils/response.py

```diff
@@ -106,7 +108,58 @@
 
         if request.method == 'POST':
             context.bind('transaction', manifest.backend.transaction, write=True)
-            data = await request.json()
+
+            # only `application/json` is supported
+            ct = request.headers.get('content-type')
+            if ct != 'application/json':
+                return JSONResponse(
+                    {"error": "only 'application/json' content-type is supported"},
+                    status_code=415
+                )
```

Comment:
You shoul dalways raise an `HTTPError` and response will be handled by http exception handler.

Reply:
Since we don't want to be too dependent on web framework, we should have our on set of http exceptions in `spinta.exceptinos`.